x-pack/plugins/monitoring/public/application/pages/elasticsearch/ingest_pipeline_modal.tsx

```diff
@@ -25,11 +25,20 @@ const INGEST_PIPELINE_DASHBOARD_ID = 'elasticsearch-metrics-ingest-pipelines';
 export const ingestPipelineTabOnClick = async (
   services: Partial<CoreStart & { dashboard: DashboardStart; fleet: FleetStart }>
 ) => {
-  const dashboard = await services.savedObjects!.client.get(
-    'dashboard',
-    INGEST_PIPELINE_DASHBOARD_ID
-  );
-  const dashboardFound = !dashboard.error && dashboard.attributes;
+  const response = await services.http!.fetch<GetBulkAssetsResponse>('/api/fleet/epm/bulk_assets', {
```

Comment:
Would it be possible to use the fleet plugin directly? We could create a service that would expose the logic in [getBulkAssetsHandler](https://github.com/elastic/kibana/blob/a2bd9a78975895a82459adfbc10bdfc627e22785/x-pack/plugins/fleet/server/routes/epm/handlers.ts#L314) and call the service from here.

We should be careful with adding http request, as there are changes in serverless that has to be adopted: https://github.com/elastic/kibana/issues/152287

Reply:
As discussed, exporting the service from `fleet/server/plugin` wouldn't allow to use it from `/public`. I'm looking how to export a hook instead, since there is already `sendGetBulkAssets` that does exactly what is needed here.